onnxruntime/python/tools/transformers/models/bart/utils/export_helper.py

```diff
@@ -0,0 +1,136 @@
+import os
+from typing import List, Tuple
+
+import torch
+from transformers import BartConfig, BartForConditionalGeneration, BartTokenizer
+from utils import export_summarization_enc_dec_past
+
+from onnxruntime import InferenceSession, SessionOptions
+
+
+class PastKeyValuesHelper:
+    """Helper functions to process past key values for encoder-decoder model"""
+
+    @staticmethod
+    def group_by_self_and_cross(present_key_values, concat=False):
```

Comment:
Is it required to be a static method? Prefer global functions in a module.

Reply:
Some of these are legacy. I will do a better clean up.